cookbook/how_to_create_new_bundle.rst

```diff
@@ -66,6 +66,13 @@ Also new bundle can be generated using `Symfony command generate:bundle`_:
 Important thing is that you shouldn't update Kernel and routing - OroPlatform provides it's own way to do that -
 it will be described in the chapter `Enable bundle`_ and in future articles.
 
```

Comment:
Edit: It is important that you don't need to update Kernel and routing, as OroPlatform provides its own way to do that, which will be described in the `Enable bundle`_ chapter and in following articles.